src/desktop/apps/auction_support/routes.coffee

```diff
@@ -109,25 +110,26 @@ registerOrRender = (sale, req, res, next) ->
         data: { access_token: req.user.get('accessToken') }
         error: res.backboneError
         success: render
-  req.user.checkRegisteredForAuction
-    saleId: sale.get('id')
-    error: res.backboneError
-    success: (registered) ->
-      res.locals.sd.REGISTERED = registered
-      render()
 
-  # TODO: Refactor all of this junk to use MP, or drop it in favor of
-  # inline bidding component, see: https://github.com/artsy/force/issues/5118
   metaphysics
     query: """ {
       artwork(id: "#{req.params.artwork}") {
         sale_artwork {
           bid_increments
         }
       }
+      me {
+        has_qualified_credit_cards
+        bidders(sale_id: "#{sale.get('id')}") {
+          id
+        }
+      }
     } """
-  .catch(next).then ({ artwork }) ->
+    req: req
+  .catch(next).then ({ artwork, me }) ->
     res.locals.bidIncrements = artwork.sale_artwork.bid_increments
+    res.locals.sd.REGISTERED = Boolean(me && me.bidders && me.bidders.length > 0)
```

Comment:
maybe `IS_REGISTERED` is better?

also ⛳️ its ☕️ so can do `me?.bidders?.length?` I believe

Reply:
gah ☕️